python/cudnn/deepseek_sparse_attention/score_recompute/sparse_score_recompute_sm100.py

```
@@ -109,27 +109,34 @@ def __init__(
         self.topk_in_smem = topk_in_smem
 
         # Pad head_dim to multiple of 16
         hdim_multiple_of = 16
-        self.head_dim_padded = int(math.ceil(head_dim / hdim_multiple_of) * hdim_multiple_of)
+        self.head_dim_padded = int(
+            math.ceil(head_dim / hdim_multiple_of) * hdim_multiple_of
+        )
         assert self.head_dim_padded % 64 == 0, (
-            f"head_dim_padded ({self.head_dim_padded}) must be a multiple of 64 " f"(K loading uses 8 threads × 8 elements = 64 per iteration)"
+            f"head_dim_padded ({self.head_dim_padded}) must be a multiple of 64 "
+            f"(K loading uses 8 threads × 8 elements = 64 per iteration)"
```

Ruff (0.15.18)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)